include/boost/detail/catch_exceptions.hpp

```diff
@@ -26,7 +26,7 @@
 #include <boost/cstdlib.hpp>  // for exit codes
 #include <ostream>         // for ostream
 
-# if defined(__BORLANDC__) && (__BORLANDC__ <= 0x0551)
+# if defined(BOOST_BORLANDC) && (BOOST_BORLANDC <= 0x0551)
```

Comment:
`#include <boost/config.hpp>` is missing. Same below.

Reply:
You are correct and I have updated the file in the PR. However the header file is using BOOST_NO_EXCEPTIONS, which definitely does require config.hpp, so should not boost/config.hpp be included ?

I also do not know what you mean by "Same below".

Reply:
No, my point is that you should add `#include <boost/config.hpp>`.

> I also do not know what you mean by "Same below".

I mean the other file you changed in the PR. It has the same issue of missing `boost/config.hpp`.

Reply:
Done !